README.md

```diff
@@ -142,3 +160,12 @@ folder.
 #### Coverage
 If coverage is installed globally, it can take wrong version inside virtualenv,
 so better uninstall it.
+
+### Release History
```

Comment:
Does it make sense to be moved to a separate file or add it to the release description https://github.com/Livit/Labster.OAuth2Client/releases ?

Reply:
- based on merit: I see no difference between Readme.md vs a separate file. Of course unless we have an established convention of using "a separate file" approach.
- personal opinion: I'd rather have 1 file to remember about doc-wise than >1 files

Reply:
As for the release description in github: I'd rather write my docs first as I work with the code, and copy-paste to GH later when releasing.

Reply:
According to https://www.makeareadme.com/ 

> A README is a text file that introduces and explains a project. It contains information that is commonly required to understand what the project is about.

Release History is not that information.
It makes no much sense to load whole release history ( imagine how big it will be in 10+ releases) each time I open repo.

Cookie cutter project suggest having [CHANGELOG.md](https://github.com/Livit/Livit.Learn.CookieCutterDjango/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/CHANGELOG.md)

Reply:
Removing the history then. No one really asked for that. I can live without that. No problem!

Reply:
Indeed. I've mentioned this to several open source projects - nothing worse than having to plough through reams of irrelevant change history before I can even find out what the project is supposed to do!

Reply:
@labstersteve obviously I agree. That is why I intended to put it _after_ the primary content of Readme :)